habitat/tests.sh

```diff
@@ -7,34 +7,28 @@
 set -eo pipefail
 
 # Execute all commands that currently work
-echo -n "Chef Inspec "
-inspec -v
-echo "Chef Infra"
-echo -n " - knife "
+# at least this is telling us if they run
+echo "> Tooling (versions)"
+chef -v
+echo "More Chef Infra"
+echo -n " - knife version: "
 knife -v | awk {'print $NF'}
-echo -n " - chef-client "
-chef-client -v | awk {'print $NF'}
-echo -n " - chef-solo "
+echo -n " - chef-solo version: "
 chef-solo -v | awk {'print $NF'}
-echo -n " - chef-apply "
+echo -n " - chef-apply version: "
 chef-apply -v | awk {'print $NF'}
-echo -n " - chef-shell "
+echo -n " - chef-shell version: "
 chef-shell -v | awk {'print $NF'}
-# TODO(afiune) do we still use this? because it doesn't work
-echo " - chef-zero (not-working)"
-#chef-zero -v
-echo -n "Berkshelf "
+echo -n "Berkshelf version: "
 berks -v
-cookstyle -v
-foodcritic -V
 ohai -v
-kitchen -v
 echo "Chef-Vault (only-help)"
 chef-vault -h
+echo "DCO (only-help)"
+dco
```

Comment:
Even though we now have `git` as part of the installation of chef-dk via Habitat package, `dco` just defaults to run `git` commands, we might need to modify the way we run those commands to find the binary, I have created a few PRs to fix this on the downstream dependencies:

https://github.com/coderanger/dco/pull/6
https://github.com/ruby-git/ruby-git/pull/416

Do we still use this gem/command?

Reply:
I'm pretty sure `dco` was abandoned and we can get rid of it - paging @tas50 to confirm

Reply:
Deprecate it would be really cool, especially since there is no movement on those gems, I could find another way to work around it but, I'll wait for confirmation from @tas50.

Reply:
> Do we still use [the dco] gem/command?

No. As @tyler-ball [mentioned](https://github.com/chef/chef-dk/pull/2209#discussion_r311791089), we've talked about (and I think decided) to pull the dco gem out of Workstation/DK.